### PR TITLE
libbsd 0.11.5

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  ==> nm /home/linuxbrew/.linuxbrew/Cellar/libbsd/0.11.5/lib/libbsd.so
  nm: /home/linuxbrew/.linuxbrew/Cellar/libbsd/0.11.5/lib/libbsd.so: File format not recognized
  
  Error: libbsd: failed
  An exception occurred within a child process:
    Minitest::Assertion: Expected: 0
    Actual: 1
```

**Comment by carlocab** [comment]
@Homebrew/linux

**Comment by iMichka** [comment]
```
➜  Formula git:(bump-libbsd-0.11.5) ✗ nm /home/linuxbrew/.linuxbrew/Cellar/libbsd/0.11.5/lib/libbsd.so
nm: /home/linuxbrew/.linuxbrew/Cellar/libbsd/0.11.5/lib/libbsd.so: file format not recognized
➜  Formula git:(bump-libbsd-0.11.5) ✗ ll /home/linuxbrew/.linuxbrew/Cellar/libbsd/0.11.5/lib/libbsd.so
-r--r--r-- 1 linuxbrew linuxbrew 196 Feb  7 23:02 /home/linuxbrew/.linuxbrew/Cellar/libbsd/0.11.5/lib/libbsd.so
➜  Formula git:(bump-libbsd-0.11.5) ✗ file /home/linuxbrew/.linuxbrew/Cellar/libbsd/0.11.5/lib/libbsd.so
/home/linuxbrew/.linuxbrew/Cellar/libbsd/0.11.5/lib/libbsd.so: ASCII text
➜  Formula git:(bump-libbsd-0.11.5) ✗ cat /home/linuxbrew/.linuxbrew/Cellar/libbsd/0.11.5/lib/libbsd.so
/* GNU ld script
* The MD5 functions are provided by the libmd library. */
OUTPUT_FORMAT(elf64-x86-64)
GROUP(/home/linuxbrew/.linuxbrew/Cellar/libbsd/0.11.5/lib/libbsd.so.0.11.5 AS_NEEDED(-lmd))
```

Let met push a fix for this

File: Formula/libbsd.rb [hunk 1]
```diff
@@ -26,6 +26,6 @@ def install
   end
 
   test do
-    assert_match "strtonum", shell_output("nm #{lib/"libbsd.so"}")
+    assert_match "strtonum", shell_output("nm #{lib/"libbsd.so.#{version}"}")
```
Comment (carlocab) [inline Formula/libbsd.rb:29]: Interpolation inside interpolation. Terrible

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**